web/app/themes/radcliffe/header.php

```diff
@@ -2,46 +2,49 @@
 
 <html <?php language_attributes(); ?>>
 
-<head profile="http://gmpg.org/xfn/11">
```

Comment:
I'm not sure why this was URI was included in the profile attribute however, please be aware that the profile attribute is still used by some legacy RDF systems. Simply removing it could cause an issue with them. Would you kindly offer your reasons for removing it?

Reply:
Ok. Reverted to original.